common/app/views/fragments/contentMeta.scala.html

```diff
@@ -22,9 +22,8 @@
 @metaBody() = {
 
     @if(item.content.showCircularBylinePicAtSide && !item.content.isImmersive) {
-        <div class="media__img meta__image">
+
```

Comment:
At this point, are we comfortable making a change that will impact the non-Garnett visitors too?

Reply:
(and I saw you added it in the template, which makes much more sense imho – no phantom tag!)

Reply:
So, this stops an empty image div from appearing on articles when they don't have one. It's a fix that that should be good for both.

Reply:
exactly!